Build.fsx

```diff
@@ -244,14 +243,13 @@ Target "PublishNuGetAll" DoNothing
 "PatchAssemblyVersions" ==> "Build"
 "BuildOnly"             ==> "Build"
 
+"BuildOnly" 
+    ==> "TestOnly"
+    ==> "BuildAndTestOnly"
+
 "Build"    ==> "Test"
 "TestOnly" ==> "Test"
 
-"BuildOnly" ==> "TestOnly"
-
-"BuildOnly" ==> "BuildAndTestOnly"
-"TestOnly"  ==> "BuildAndTestOnly"
-
```

Comment:
I'm aware that they have been around for a while; I figured that, since we're refactoring the build script, we might as well do some spring cleaning while we're at it. 😉

I see your point about them possibly being useful to someone. Let's keep them.